index.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+
+$db = new PDO('mysql:host=db; dbname=ukgamesfunded', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare("SELECT `title`, `year` , `theme` , `genre` , `rating`  FROM `game`");
+
+$query->execute();
+
+$result = $query->fetchAll();
```

Comment:
I would also suggest putting these three lines into another re-usable function. The same sort of idea as my last comment, if you ever needed to get a list of your collection items again you could simply call the function rather than having to copy and paste these lines.

Think about getting data into and out of function, remember the only way to get data into a function is using parameters, and the only way to get data out of a function is by returning it. So think about what you might need to get into the function in order for it to do it's job - let me know if you'd like some clues!